### ISSUE TITLE
.email() does not respect UTF8SMTP Extension standard (which includes öäü aka. Umlaute)

### ISSUE THREAD

**Issue by manuel-rw** [colinhacks/zod#3073]
Hello,
the RFC standard https://www.rfc-editor.org/rfc/rfc5336#section-3.2 allows users to use "öäü" and other special characters as part of their Email.
Further information here: https://stackoverflow.com/questions/15121359/are-international-characters-e-g-umlaut-characters-valid-in-the-local-part-of

``.email()`` currently does not respect this nor does it provide the option to allow this extension specifically.

Therefore, the following email will fail unexpectedly: ``schäfer2@gmail.com``

Via https://github.com/ajnart/homarr/issues/1747

**Comment by maybeanerd** [comment]
@JacobWeisenburger  this is not an enhancement, but a bug IMO
As it incorrectly fails valid emails

**Comment by maybeanerd** [comment]
Is anybody working on this right now? If not I might try picking it up this weekend

**Comment by manuel-rw** [comment]
I would assume no. We are waiting for the upstream fix and sadly do not have capacity at the current time to fix this.

**Comment by scotttrinh** [comment]
I agree this is an enhancement due to this being about supporting an extension. The built-in validators are provided as best-efforts, but like in the case of numbers being valid emojis, your use case might not be covered by the existing check. Feel free to use `string().regex()` with your own email regex if you feel stuck, or submit a PR: I'd be happy to review something that passed the existing tests and adds support for this extension. We might want to make this a separate email type (`.emailUtf8`?) to support users who do not support the UTF8SMTP extension (which is an experimental standard as best I can tell that has been superceded by [RFC 6531](https://www.rfc-editor.org/rfc/rfc6531#section-3.2)).

**Comment by maybeanerd** [comment]
Sorry, life happened. I haven't started work on this , somebody else feel free to do so. If I get to it, I'll comment on this thread to let you know.

**Comment by simonmaass** [comment]
this would be great!

**Comment by gshpychka** [comment]
> What is the status of this?

Out of scope: https://github.com/colinhacks/zod/pull/2157

**Comment by colinhacks** [comment]
Correct. Zod's policy on email validation is now long established. https://colinhacks.com/essays/reasonable-email-regex
